src/local_sensitivity/backsolve_adjoint.jl

```diff
@@ -112,15 +127,14 @@ end
 @noinline function SDEAdjointProblem(sol,sensealg::BacksolveAdjoint,
                                      g,t=nothing,dg=nothing;
                                      checkpoints=sol.t,
-                                     callback=CallbackSet(),kwargs...)
+                                     callback=CallbackSet(),
+                                     diffusion_jac=nothing, diffusion_paramjac=nothing,kwargs...)
```

Comment:
assume these will never be called directly. These would need to be added to the `SDEFunction` to be used.

Reply:
to the `SDEFunction` or `ODEFunction`? 
They are indeed not yet tested (or used and could be left out for the moment) -- I thought that something like this might be a nice way to encode if one of the two Jacobians dsigma/dx or dsigma/dp is zero or constant (to speed up the computation).